continuous_integration/recipes/distributed/meta.yaml

```
@@ -3,79 +3,104 @@
 {% set version = (major_minor_patch[:2] + [new_patch]) | join('.') + environ.get('VERSION_SUFFIX', '') %}
 {% set dask_version = environ.get('DASK_CORE_VERSION', '0.0.0.dev') %}
 
+{% set proc_version = "1.0.0" %}
+{% set proc_build_number = "0" %}
+{% set cython_enabled = build_ext == "cython" %}
+
 
 package:
-  name: distributed
+  name: distributed-split
   version: {{ version }}
 
 source:
   git_url: ../../..
 
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: py{{ python | replace(".", "") }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
-  script: {{ PYTHON }} -m pip install . -vv
-  entry_points:
-    - dask-scheduler = distributed.cli.dask_scheduler:go
-    - dask-ssh = distributed.cli.dask_ssh:go
-    - dask-worker = distributed.cli.dask_worker:go
-
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-  host:
-    - python
-    - pip
-
-  run:
-    - python
-    - click >=6.6
-    - cloudpickle >=1.5.0
-    - cytoolz >=0.8.2
-    - dask-core >={{ dask_version }}
-    - jinja2
-    - msgpack-python >=0.6.0
-    - packaging >=20.0
-    - psutil >=5.0
-    - pyyaml
-    - sortedcontainers !=2.0.0,!=2.0.1
-    - tblib >=1.6.0
-    - toolz >=0.8.2
-    - tornado >=5  # [py<38]
-    - tornado >=6.0.3  # [py>=38]
-    - zict >=0.1.3
-    - setuptools <60.0.0
-
-  run_constrained:
-    - openssl !=1.1.1e
 
-test:
-  imports:
-    - distributed
-    - distributed.cli
-    - distributed.comm
-    - distributed.deploy
-    - distributed.diagnostics
-    - distributed.protocol
-  commands:
-    - pip check
-    - dask-scheduler --help
-    - dask-ssh --help
-    - dask-worker --help
-  requires:
-    - pip
+outputs:
+  - name: distributed-proc
+    version: {{ proc_version }}
+    build:
+      number: {{ proc_build_number }}
+      string: {{ build_ext }}
```

As this should be architecture agnostic, we can use `noarch` here

```suggestion
      string: {{ build_ext }}
      noarch: generic
```